src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -406,6 +406,7 @@
  * &lt;module name=&quot;ImportOrder&quot;&gt;
  *   &lt;property name=&quot;staticGroups&quot; value=&quot;org,java&quot;/&gt;
  *   &lt;property name=&quot;sortStaticImportsAlphabetically&quot; value=&quot;true&quot;/&gt;
+ *   &lt;property name=&quot;option&quot; value=&quot;top&quot;/&gt;
```

Comment:
it is strange how it result in bad rendering

![image](https://user-images.githubusercontent.com/812984/122683598-25e54400-d1b5-11eb-838d-47a2b4b34d7c.png)